README.md

```diff
@@ -62,30 +65,35 @@ jobs:
       contents: read # Needed to checkout repositories
       issues: write # Needed if you set "file_issue: true"
       id-token: write # Needed to detect the current reusable repository and ref
-    uses: sigstore/rekor-monitor/.github/workflows/reusable_monitoring.yaml@main
-    with:
-      file_issue: true # Strongly recommended: Files an issue on monitoring failure
-      artifact_retention_days: 14 # Optional, default is 14: Must be longer than the cron job frequency
-      identities: |
-        certIdentities:
-          - certSubject: user@domain\.com
-          - certSubject: otheruser@domain\.com
-            issuers:
-              - https://accounts\.google\.com
-              - https://github\.com/login
-          - certSubject: https://github\.com/actions/starter-workflows/blob/main/\.github/workflows/lint\.yaml@.*
-            issuers:
-              - https://token\.actions\.githubusercontent\.com
-        subjects:
-          - subject@domain\.com
-        fingerprints:
-          - A0B1C2D3E4F5
-        fulcioExtensions:
-          build-config-uri:
-            - https://example.com/owner/repository/build-config.yml
-        customExtensions:
-          - objectIdentifier: 1.3.6.1.4.1.57264.1.9
-            extensionValues: https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v1.4.0
+    uses: sigstore/rekor-monitor/.github/workflows/identity_monitor.yaml@main
+```
+
+```
+monitoredValues: |
```

Comment:
Does this need to be input to the workflow? Or written to a file before running the workflow?

Reply:
Written to a file and filepath is passed in, updated README to reflect that

Reply:
Can you demonstrate this in the yaml as [a run step](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) before `uses`? Something like:

```
run: |
  s=<yaml string>
  cat s > id_file_path.yaml
```

Give a full example with the yaml. Let's also make sure this is tested. You'll need to set up a repository like https://github.com/haydentherapper/rekor-log-monitor to add a workflow and test your example works.